examples/ReadSensorsImperial/ReadSensorsImperial.ino

```diff
@@ -0,0 +1,68 @@
+/*
+  MKR ENV Shield - Read Sensors Imperial
+
+  This example reads the sensors on-board the MKR ENV shield
+  and prints them in imeprial units to the Serial Monitor once a second.
+
+  The circuit:
+  - Arduino MKR board
+  - Arduino MKR ENV Shield attached
+
+  This example code is in the public domain.
+*/
+
+#include <Arduino_MKRENV.h>
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial);
+
+  if (!ENV.begin()) {
+    Serial.println("Failed to initialize MKR ENV shield!");
+    while (1);
+  }
+}
+
+void loop() {
+  // Seting to FAHRENHEIT and FOOTCANDLE readTemperature and readIlluminance
```

Comment:
```suggestion
  // Setting readTemperature to FAHRENHEIT and readIlluminance to FOOTCANDLE
```

Reply:
I think this should be reworded to pass in `FAHRENHEIT` as the units argument ...

Reply:
done like as suggested by @sandeepmistry